server/Pipfile

```diff
@@ -7,6 +7,12 @@ verify_ssl = true
 
 [packages]
 flask = "*"
+sqlalchemy = "*"
+psycopg2 = "*"
+flask-sqlalchemy = "*"
+flask-migrate = "*"
+flask-script = "*"
+python-dotenv = "*"
```

Comment:
I feel like we should start putting version numbers here, @aeciorc thoughts?

Reply:
flask-script was [deprecated](https://github.com/smurfix/flask-script) two years ago. flask-migrate supports its own way of issuing CLI commands; do we need an extra deprecated dependency just so we can say `python manage.py db migrate` instead of `flask db migrate`?

Reply:
@skassam21 Have pinned versions in the newest commit.

@thaumasiotes I was not familiar with flask-migrate and blindly followed the outdated tutorial. 😅I have removed flask-script from Pipfile and verified `init` `migrate` and `upgrade` works.